Makefile

```
@@ -1,13 +1,18 @@
-sources = isrgrootx1.signing_policy.txt \
-          letsencryptauthorityx3.signing_policy.txt \
-          letsencryptauthorityx4.signing_policy.txt
+sources = isrgrootx1.signing_policy \
+          letsencryptauthorityx3.signing_policy \
+          letsencryptauthorityx4.signing_policy \
+	  lets-encrypt-r3.signing_policy \
+	  lets-encrypt-r4.signing_policy \
 
 targets = 23c2f850.signing_policy 4042bcee.signing_policy \
           4a0a35c0.signing_policy 4f06f81d.signing_policy \
           6187b673.signing_policy 929e297e.signing_policy \
-          23c2f850.0 4042bcee.0 4a0a35c0.0 4f06f81d.0 6187b673.0 929e297e.0 \
-          isrgrootx1.pem.txt \
-          letsencryptauthorityx3.pem.txt letsencryptauthorityx4.pem.txt
+	  8d33f237.signing_policy dec71a0b.signing_policy \
+	  9f194ecd.signing_policy dd7d39a7.signing_policy \
+          23c2f850.0 4042bcee.0 4a0a35c0.0 4f06f81d.0 6187b673.0 8d33f237.0 929e297e.0 9f194ecd.0 dec71a0b.0 dd7d39a7.0 \
+          isrgrootx1.pem \
+          letsencryptauthorityx3.pem letsencryptauthorityx4.pem \
+	  lets-encrypt-r3.pem lets-encrypt-r4.pem
```

Whitespace, and respect upstream's attempt to preserve 80-wide here

```suggestion
          8d33f237.signing_policy dec71a0b.signing_policy \
          9f194ecd.signing_policy dd7d39a7.signing_policy \
          23c2f850.0 4042bcee.0 4a0a35c0.0 4f06f81d.0 6187b673.0 \
          8d33f237.0 929e297e.0 9f194ecd.0 dec71a0b.0 dd7d39a7.0 \
          isrgrootx1.pem \
          letsencryptauthorityx3.pem letsencryptauthorityx4.pem \
          lets-encrypt-r3.pem lets-encrypt-r4.pem
```

---

I've split this up slightly differently to group the relevant hashes together